lightning/src/ln/mod.rs

```diff
@@ -96,7 +96,7 @@ mod async_signer_tests;
 #[cfg(test)]
 #[allow(unused_mut)]
 mod offers_tests;
-#[allow(dead_code)] // TODO(dual_funding): Exchange for dual_funding cfg
+#[allow(dead_code)] // TODO(splicing): Exchange for splicing cfg
```

Comment:
I think the dead_code exception no longer needed. This change seems to be search/and/replace false positive.

Reply:
Just checked. Is still needed for now :(